experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -330,6 +330,9 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
       'response',
       (response: http.IncomingMessage & { aborted?: boolean }) => {
         this._diag.debug('outgoingRequest on response()');
+        if (request.listenerCount('request') <= 0) {
```

Comment:
```suggestion
        if (request.listenerCount('response') <= 0) {
```

Same naive/untested Q as on the equivalent PR for the "next" branch: Is the eventName here meant to be "response"?

Reply:
wow yes it is

Reply:
It's only a problem is the actual user application starts data receiving late for some reason or decides to read manually from stream instead by registering a `data` listener or pipe into another stream.
But well, current tests are not doing this.